json/functions.php

```diff
@@ -213,6 +213,30 @@ public function getMediaAttachments( $post ) {
 		$post->media_attachments 	= $attachments_filtered;
 		return $post;
 	}
+  
+  /** 
+   * @desc Returns the featured image of a post, if set by user.
+   * @param object $post -- The Post from the loop.
+   * returns object if true, boolean if no featured post set
+   */
+  
+  public function getThumbnail ( $post ) {
```

Comment:
Call this method `getPostThumbnail` to give it better context and optionally pass in the size:
```php
public function getPostThumbnail ( $post, $thumbnail_size="large" ) {
```

Also, use tabs instead of spaces for indentation so the white space in your contribution is consistent with the rest of the code base. If you are using Sublime Text, you can add the following settings to reveal how white space is being rendered and clean up excessive white space: 
```json
	"draw_white_space": "all",
	"trim_trailing_white_space_on_save": true,
	"trim_automatic_white_space": true
```